packages/mobile/test/schemas.ts

```diff
@@ -592,6 +592,22 @@ export const v9Schema = {
   },
 }
 
+export const v10Schema = {
+  ...v9Schema,
+  identity: {
+    ..._.omit(
+      v9Schema.identity,
+      'attestationCodes',
+      'acceptedAttestationCodes',
+      'attestationInputStatus',
+      'numCompleteAttestations',
+      'verificationStatus',
+      'hasSeenVerificationNux',
+      'lastRevealAttempt'
+    ),
+  },
+}
```

Comment:
`verify` key should be moved here from https://github.com/celo-org/wallet/pull/377/files#diff-a0850796be5a07a6363631e739d56ea69fa51c26852655610c6f7650285b9a2e

Reply:
and `identity` keys shouldn't have been removed from `v9Schema` in https://github.com/celo-org/wallet/pull/377/files#diff-a0850796be5a07a6363631e739d56ea69fa51c26852655610c6f7650285b9a2e